app/src/main/res/layout/resume_lesson_activity.xml

```diff
@@ -24,7 +24,7 @@
         app:navigationIcon="?attr/homeAsUpIndicator"
         app:title="@string/resume_lesson_activity_title"
         app:titleTextAppearance="@style/ToolbarTextAppearance"
-        app:titleTextColor="@color/color_def_white" />
+        app:titleTextColor="@color/component_color_shared_activity_toolbar_text_color" />
```

Comment:
This can be removed as we are using `ToolbarTextAppearance ` which already contains this color.

Reply:
Also, use shared-toolbar-color-code for background

Reply:
Done.